pkg/minikube/config/profile.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/golang/glog"
+	"k8s.io/minikube/pkg/minikube/constants"
+)
+
+// AllProfiles returns all minikube profiles
+func AllProfiles() (profiles []*Profile) {
```

Comment:
Unless there is a different function that returns a subset of profiles, I would rather see this function named ListProfiles or Profiles.